API-reference.md

```diff
@@ -414,7 +414,7 @@ Sends telemetry to the endpoint.
 
 ### addTelemetryInitializer
 
-        public addTelemetryInitializer(telemetryInitializer: (envelope: Telemetry.Common.Envelope) => boolean)
+        public addTelemetryInitializer(telemetryInitializer: (envelope: Telemetry.Common.Envelope) => boolean | void)
```

Comment:
@KamilSzostak - docs still not working with definition types,  Microsoft.ApplicationInsights.IEnvelope doesn't have member data...